packages/smooth_app/lib/pages/product/common/product_query_page.dart

```diff
@@ -327,18 +327,16 @@ class _ProductQueryPageState extends State<ProductQueryPage> {
     if (refreshSupplier == null) {
       return;
     }
-    final String lastTime =
+    final String lastTimeMessage =
         ProductQueryPageHelper.getDurationStringFromTimestamp(
             _lastUpdate, context);
     _lastUpdate = null;
-    final String message =
-        '${AppLocalizations.of(context).cached_results_from}: $lastTime.';
 
     Future<void>.delayed(
       const Duration(seconds: 0),
       () => ScaffoldMessenger.of(context).showSnackBar(
```

Comment:
@monsieurtanuki, is there a reason why we make .delayed with a duratin of 0

Reply:
Could have been for historical reasons - before flutter 2.0 displaying snackbars was sometimes problematic.
Feel free to test without `delayed` and to remove it if possible.

Reply:
Still broken